### PR TITLE
Temporarily removed support for the secure requirements during the bu…

### PR DESCRIPTION
**Pull request by jotes**
@mathjazz I'll remove this hack when the issue with pip8 on the readthedocs.org will be resolved https://github.com/rtfd/readthedocs.org/issues/1843

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,8 @@
--r docs/requirements.txt
+# Temporarily we had to remove support for the secure dependencies, because
+# readthedocs.org don't support pip8.
+# More info at https://github.com/rtfd/readthedocs.org/issues/1843
+
+-r docs/safe_requirements.txt
```
Comment (mathjazz) [inline requirements.txt]: You mean `docs/requirements.txt`, right?

**Comment by mathjazz** [comment]
Nice work, r+wc.

File: requirements.txt (hunk 1 above)
Comment (jotes) [inline requirements.txt]: I wanted to leave safe requirements for developers. I can remove safe_requirements file and just leave docs/requirements.txt if that's what you suggest.

Comment (mathjazz) [inline requirements.txt]: Whoops, that's the main requirements file, my bad!

**Pull request merged**